scripts/NewTeamsAlert/src/utils/filepath.js

```diff
@@ -0,0 +1,4 @@
+
+const TODO_FILE_PATH = "/Users/parth.maradia/Project/Intern Project/Comments-Manager/scripts/codeOptimizer/data/todo_new.json"
```

Comment:
this file path can be different for based on where code is residing how are going to handle that?